Connectivity/computeConductanceFromTensors.m

```diff
@@ -14,11 +14,20 @@ function computeConductanceFromTensors(data_folder)
 %%% Prepare inputs
```

Comment:
I actually forgot to delete this file. It is as the one in the examples folder (run_conductance_model.m) but the example is updated. Sorry about that!